components/sso-saml/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/validators/IdPInitSSOAuthnRequestValidator.java

```diff
@@ -60,7 +60,8 @@ public SAMLSSOReqValidationResponseDTO validate() throws IdentityException {
                 validationResponse.setValid(false);
                 String errorResp = SAMLSSOUtil.buildErrorResponse(
                         SAMLSSOConstants.StatusCodes.REQUESTOR_ERROR,
-                        "spEntityID parameter not found in request");
+                        "spEntityID parameter not found in request",
+                        null);
```

Comment:
Changed to utilize full line length in https://github.com/wso2/carbon-identity/pull/685/files#diff-b21d6d7b652e46346fad773c1400e373R59